packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -432,9 +437,11 @@ export class SDKClient {
       ethereumTransactionData.callData = new Uint8Array();
       ethereumTransaction.setEthereumData(ethereumTransactionData.toBytes()).setCallDataFileId(fileId);
     }
+    const networkGasPriceInTinyBars = weibarHexToTinyBarInt(networkGasPriceInWeiBars)!;
```

Comment:
QQ: In which cases is this going to be `null` or `undefined`?

Reply:
The `weibarHexToTinyBarInt(value)` returns `null` if value is `null` | `undefined` or the value is an empty hash, `0x`.

Reply:
In this case let’s handle this more explicitly by doing a check instead of using `!` here:
```typescript
    const networkGasPriceInTinyBars = weibarHexToTinyBarInt(networkGasPriceInWeiBars);
    if (!networkGasPriceInTinyBars) {
      throw predefined.SOME_RELEVANT_ERROR_HERE;
    }
```

Reply:
Yeah that's a good point. I originally thought since the `networkGasPriceInWeiBars` was a number and a `predefine.COULD_NOT_ESTIMATE_GAS_PRICE` has already been thrown but you're right we should also handle the error here as well. Pushing in the fix soon.